packages/material-react-table/src/components/table/MRT_TablePaper.tsx

```diff
@@ -58,7 +61,7 @@ export const MRT_TablePaper = <TData extends MRT_RowData>({
               right: 0,
               top: 0,
               width: '100dvw',
-              zIndex: 999,
+              zIndex: theme.zIndex.modal + 1,
```

Comment:
@helt, please don't add "+1" to theme.zIndex.modal. This will cause, other modal and dialog components to appear below MRT_TablePaper

Reply:
@sadik-malik Whats your suggestion to use then? Plain `theme.zIndex.modal` ?

Reply:
yes, if it is not breaking any of the test cases.

Reply:
https://stackblitz.com/edit/github-morytr-zqdrka?file=src%2Fexample.tsx

@helt, In the above stackblitz example you can see Dialog component is appearing below MRT_TablePaper when zIndex is set to `theme.zIndex.modal + 1`

Reply:
@helt any update

Reply:
so this should work perfectly fine as long as no `+ 1` is added?